Packet++/header/IcmpLayer.h

```diff
@@ -3,6 +3,7 @@
 
 #include "Layer.h"
 #include "IPv4Layer.h"
+#include "IpAddress.h"
```

Comment:
is this include actually needed? 

`IPv4Layer.h` is already including `IpAddress.h`

Reply:
An include has been removed